README.md

```diff
@@ -21,8 +21,3 @@
   <img src="https://github.com/microsoft.png" width="40px" height="40px" style="border-radius:50%"  />
   <a href="https://github.com/microsoft">Microsoft</a>
 </p>
-
-<p style="display: flex; align-items:center; gap:10px;">
-  <img src="https://github.com/SulaimonShittu.png" width="40px" height="40px" style="border-radius:50%"  />
-  <a href="https://github.com/SulaimonShittu">Sulaimon Shittu</a>
-</p>
```

Comment:
This PR is removing this user data, rather you should sync update with your forked version before making a PR